### PR TITLE
pin golint for release 1.4

### PR DESCRIPTION
**Pull request by jessfraz**
Fixes https://github.com/kubernetes/kubernetes/issues/37254 for the release-1.4 branch

(I pushed the artifact)

### CODE REVIEW

**Comment by k8s-oncall** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1239)

**Comment by jessfraz** [comment]
I am testing I picked the correct golint version and testing the new image here: https://github.com/kubernetes/kubernetes/pull/37710

**Comment by jessfraz** [comment]
so nevermind I would need to merge this to make sure https://github.com/kubernetes/kubernetes/pull/37710 works, is that ok?

**Comment by jessfraz** [comment]
testing locally

**Comment by jessfraz** [comment]
sweet that image works locally so ping @ixdy for the merge please :)

**Comment by krzyzacy** [comment]
I wonder do we need to pin a version for master/1.5 as well

**Comment by jessfraz** [comment]
that is being discussed in https://github.com/kubernetes/kubernetes/issues/37254 but pinning for a release branch is sane

File: jenkins/test-image/Dockerfile-1.4 [hunk 1]
```diff
@@ -48,8 +48,13 @@ RUN apt-get -o Acquire::Check-Valid-Until=false update && apt-get install -y \
...
-RUN go get \
-        github.com/golang/lint/golint
+# Get dependency for golint
+ENV GO_TOOLS_COMMIT b66e054640c8249b6bc92cebffaec2bcf8b5efd0
+RUN git clone https://github.com/golang/tools.git /go/src/golang.org/x/tools \
+	&& (cd /go/src/golang.org/x/tools && git checkout -q $GO_TOOLS_COMMIT)
 
+# Get Go's lint tool
+ENV GO_LINT_COMMIT 64229b89515c2a585c623c79a7ccdea71e8589ff
+RUN git clone https://github.com/golang/lint.git /go/src/github.com/golang/lint \
```
Comment (jessfraz) [inline jenkins/test-image/Dockerfile-1.4]: ya then i put the tools on the same `RUN` line, will update

**Comment by jessfraz** [comment]
updated and pushed

**Comment by ixdy** [comment]
/lgtm

Do you have permissions on this repo, or do you need one of us to merge?

**Comment by jessfraz** [comment]
Got it :) thanks

**Pull request merged**